src/app/app.ts

```diff
@@ -0,0 +1,28 @@
+import type { BaseComponent } from '@components/base-component';
+
+import { PageWrapper } from './page';
+
+class App {
+  constructor(
+    private pageWrapper: BaseComponent,
+    private root: HTMLElement,
+  ) {}
+
+  public stop(): void {
+    this.root.append(this.pageWrapper.getNode());
+  }
+
+  public start(): void {
+    this.root.append(this.pageWrapper.getNode());
+  }
+
+  public pause(): void {
+    this.root.append(this.pageWrapper.getNode());
+  }
+}
```

Comment:
Methods start, stop, and pause contain duplicated logic.
```suggestion
private renderPage(): void {
  this.root.append(this.pageWrapper.getNode());
}

public stop(): void {
  this.renderPage();
}

public start(): void {
  this.renderPage();
}

public pause(): void {
  this.renderPage();
}

```